Cross-compil from Linux to Windows failed

When building my application which use systemstat from a Linux host to a windows target (`cargo build --target=x86_64-pc-windows-gnu ` the linker failed (` /usr/lib/gcc/x86_64-w64-mingw32/11.2.0/../../../../x86_64-w64-mingw32/bin/ld: cannot find -lIphlpapi`).
The problem is that the library in mingw is in totally in lowercase (iphlpapi) but the linker attribute (in socket.rs and network_interfaces.rs) has the first letter in uppercase (Iphlpapi).
Could you change the linker attributes to make the build succeed.
It shouldn't change the Windows behavior because the filename are case insensitive in Windows.

Regards

--- a/src/platform/windows/network_interfaces.rs
+++ b/src/platform/windows/network_interfaces.rs
@@ -87,7 +87,7 @@ struct IpAdapterAddresses {
 // https://msdn.microsoft.com/en-us/library/aa365915(v=vs.85).aspx
 // https://msdn.microsoft.com/zh-cn/library/windows/desktop/aa366066(d=printer,v=vs.85).aspx
 // C:\Program Files (x86)\Windows Kits\8.1\Include\um\IPHlpApi.h
-#[link(name = "Iphlpapi")]
+#[link(name = "iphlpapi")]
 extern "system" {
     fn GetAdaptersAddresses(
         family: ULONG,

--- a/src/platform/windows/socket.rs
+++ b/src/platform/windows/socket.rs
@@ -37,7 +37,7 @@ struct UdpStats {
     num_addrs: c_ulong,
 }
 
-#[link(name = "Iphlpapi")]
+#[link(name = "iphlpapi")]
 extern "system" {
     // https://msdn.microsoft.com/en-us/library/aa366023(v=vs.85).aspx
     fn GetTcpStatisticsEx(pStats: *mut TcpStats, dwFamily: c_ulong) -> c_ulong;

